src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx

```diff
@@ -0,0 +1,142 @@
+
+## Create multiple accounts [#multiple-accounts]
+
+There are many reasons to create multiple accounts within your organization. Here are some common examples that can help you decide if a multi-account structure is right for your team.
+
+### Use multiple accounts to separate testing and production environments [#environments]
+
+Multiple accounts let you observe data across multiple environments so you can monitor changes before they go to production. If you’re updating your application regularly, it's important that you have a strong testing environment in place so you can make sure each deployment is error free.
+
+<img
+    style={{ align: 'left',maxWidth: '100%' }}
+    title="multiple environments"
+    alt="A diagram demonstrating how to use New Relic with multiple environments"
+    src={accountsMultipleAccountEnvironments}
+/>
+
+### Separate projects with unique data sets and multiple accounts [#data-sets]
+
+An organization is a container that holds different data for different accounts. Accounts can be a good way to keep data separate so that you can easily filter and query based on specific requirements. 
+
+Here are some scenarios to use multiple accounts:
+* You manage a payment provider that works in multiple countries and don't want the data (currencies, tax regulations, etc.) from each country to mix with the others. 
+* You manage a social media site and need to comply with different data collection policies in different regions.
+
+<img
+    style={{ align: 'left',maxWidth: '100%' }}
+    title="Multiple data sets"
+    alt="A diagram showing how to use New Relic with multiple data sets"
+    src={accountsUniqueDataSets}
+/>
+
+### Use accounts to manage large data limits [#large-data-limits]
+
+If you plan on ingesting vast amounts of data, it's a good idea to create multiple accounts to avoid data limits. Financial institutions and healthcare providers often ingest high volumes of data because they track transactions and user information across multiple states and at high rates. Large retailers also benefit from creating multiple accounts. If you are a single organization and with several different storefronts, creating an account for each company could be a good idea. 
+
+Here are some important limits to keep in mind:
+
+* Alert conditions default to 4,000 conditions per account.
+* Alert workflows default to 4,000 workflows per account.
+* Alert policies default 10,000 policies per account.
+* Alert destinations default 2,000 destinations per account. 
+* NRDB query inspected count limits are different with standard and data plus accounts.
+
+To learn more about limits, see:
+*  [Inspected data points for NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries/#inspected-count-limits)
+* [Alerting limits](docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/).
```

Comment:
Nitpicky but as a fast follow I'd remove this `.`

Reply:
Fixed :)